apps/src/code-studio/assets/SpriteUpload.jsx

```diff
@@ -7,22 +7,42 @@ import {
   uploadSpriteToAnimationLibrary,
   uploadMetadataToAnimationLibrary
 } from '@cdo/apps/assetManagement/animationLibraryApi';
+import classNames from 'classnames';
 
 const SpriteLocation = makeEnum('library', 'level');
 const UploadStatus = makeEnum(
   'fail',
   'success',
   'filenameOverride',
-  'badFilename',
+  'badFile',
   'none'
 );
+const TestStatus = makeEnum('pending', 'pass', 'fail', 'warn');
+const StatusIconClasses = {
```

Comment:
In the interest of consistency across the codebase, is this something we could use ValidationStep.jsx for?